locales/templates/app.pot

```diff
@@ -0,0 +1,622 @@
+msgid "SD card"
+msgstr ""
+
+msgctxt "setting_dialog_internal_storage"
+msgid "Internal storage"
+msgstr ""
+
+#. BrowsingHistoryActivity
+msgctxt "browsing_history_clear_btn"
+msgid "CLEAR BROWSING HISTORY"
+msgstr ""
+
+msgctxt "browsing_history_empty_view_msg"
+msgid "No history"
+msgstr ""
+
+msgctxt "browsing_history_date_today"
+msgid "Today"
+msgstr ""
+
+msgctxt "browsing_history_date_yesterday"
+msgid "Yesterday"
+msgstr ""
+
+msgctxt "browsing_history_menu_delete"
+msgid "Delete"
+msgstr ""
+
+msgctxt "browsing_history_dialog_confirm_clear_message"
+msgid "Clear all browsing history?"
+msgstr ""
+
+msgctxt "browsing_history_dialog_btn_clear"
+msgid "Clear"
+msgstr ""
+
+msgctxt "remove_from_list"
+msgid "Remove from list"
+msgstr ""
+
+msgctxt "delete_file"
+msgid "Delete file"
+msgstr ""
+
+#. Long Screenshot
+msgctxt "screenshot_saved"
+msgid "Screenshot saved"
+msgstr ""
+
+msgctxt "screenshot_failed"
+msgid "Failed to capture screenshot"
+msgstr ""
+
+msgctxt "screenshot_illustration"
+msgid "capturing screenshot of entire page…"
+msgstr ""
+
+#. Snack bar message
+msgctxt "message_fallback_save_to_primary_external"
+msgid "No SD card detected. File was saved to internal storage."
+msgstr ""
+
+msgctxt "message_start_to_save_to_removable_storage"
+msgid "File saved to SD card"
+msgstr ""
+
+msgctxt "message_removable_storage_space_not_enough"
+msgid "SD card is almost full. File was saved to internal storage."
+msgstr ""
+
+#. ScreenshotGridFragment
+msgctxt "screenshot_grid_empty_text_title"
+msgid "No screenshots"
+msgstr ""
+
+msgctxt "screenshot_grid_empty_text_msg_prefix"
+msgid "Tap "
+msgstr ""
+
+msgctxt "screenshot_grid_empty_text_msg_postfix"
+msgid ""
+"  on the menu to capture\n"
+"an entire web page"
+msgstr ""
+
+msgctxt "screenshot_image_viewer_dialog_title"
+msgid "Info"
+msgstr ""
+
+msgctxt "screenshot_image_viewer_dialog_info_time"
+msgid "Time:"
+msgstr ""
+
+msgctxt "screenshot_image_viewer_dialog_info_resolution"
+msgid "Resolution:"
+msgstr ""
+
+msgctxt "screenshot_image_viewer_dialog_info_size"
+msgid "Size:"
+msgstr ""
+
+msgctxt "screenshot_image_viewer_dialog_info_title"
+msgid "Title:"
+msgstr ""
+
+msgctxt "screenshot_image_viewer_dialog_info_url"
+msgid "URL:"
+msgstr ""
+
+msgctxt "screenshot_image_viewer_dialog_delete_msg"
+msgid "Do you want to delete the image?"
+msgstr ""
+
+#. Title for file picker when user trying to upload a file
+msgctxt "file_picker_title"
+msgid "Choose a file to upload"
+msgstr ""
+
+#. Toast message
+msgctxt "message_enable_turbo_mode"
+msgid "Turbo mode enabled"
+msgstr ""
+
+msgctxt "message_disable_turbo_mode"
+msgid "Turbo mode disabled"
+msgstr ""
+
+msgctxt "message_enable_block_image"
+msgid "Now blocking images"
+msgstr ""
+
+msgctxt "message_disable_block_image"
+msgid "Now showing images"
+msgstr ""
+
+msgctxt "message_clear_cache_fail"
+msgid "Unable to clear cache"
+msgstr ""
+
+#, c-format
+msgctxt "message_cleared_cached"
+msgid "%1$s cache cleared"
```

Comment:
What's the variable here? Please explaining in a comment

Reply:
Will do update the comment.
It's the amount of the cache cleared.